cmd/kyma/deploy/cmd.go

```diff
@@ -209,9 +210,15 @@ func (cmd *command) Run() error {
 	}
 
 	// TODO remove this block when default component values are migrated to kyma 2.0
+	overridesStep := cmd.NewStep("Applying Kyma2 overrides")
 	if err := overrides.AddFile(filepath.Join(cmd.opts.WorkspacePath, kyma2OverridesPath)); err != nil {
-		return errors.Wrap(err, "Could not add overrides for Kyma 2.0")
+		if errors.Is(err, fs.ErrNotExist) {
+			overridesStep.LogErrorf("Kyma2 override path not found but continuing: %s", err)
```

Comment:
It should be logged only in verbose mode as it is nothing important if it cannot be found. The overall goal is to get rid of the file